test/functional/apps/visualize/group3/_annotation_listing.ts

```diff
@@ -0,0 +1,183 @@
+        });
+
+        it('matches on description', async function () {
+          await listingTable.searchForItemWithName('i have a description');
+          await listingTable.expectItemsCount('eventAnnotation', 1);
+        });
+      });
+
+      describe('by tag', () => {
+        it('filters by tag', async () => {
+          await listingTable.selectFilterTags('tag');
+          await listingTable.expectItemsCount('eventAnnotation', 7);
+        });
+      });
+    });
+
+    describe('delete', function () {
+      it('deletes some groups', async function () {
+        await listingTable.deleteItem('to delete 1');
+        await listingTable.deleteItem('to delete 2');
+        await listingTable.searchForItemWithName('to delete');
+        await listingTable.expectItemsCount('eventAnnotation', 0, 1000);
+        await listingTable.clearSearchFilter();
+      });
+    });
+
+    describe('edit', function () {
+      it('edits group metadata', async function () {
+        await listingTable.clickItemLink('eventAnnotation', 'group 3');
+        await PageObjects.annotationEditor.editGroupMetadata({
+          title: 'edited title',
+          description: 'edited description',
+        });
+        await PageObjects.annotationEditor.saveGroup();
+
+        await listingTable.searchForItemWithName('edited title');
+        await listingTable.expectItemsCount('eventAnnotation', 1);
+
+        await listingTable.searchForItemWithName('edited description');
+        await listingTable.expectItemsCount('eventAnnotation', 1);
+      });
+
+      describe('individual annotations', () => {
+        it('edits an existing annotation', async function () {
+          await listingTable.clickItemLink('eventAnnotation', 'edited title');
+          expect(await PageObjects.annotationEditor.getAnnotationCount()).to.be(1);
+
+          await PageObjects.annotationEditor.openAnnotation();
+          await PageObjects.annotationEditor.configureAnnotation({
+            query: 'my query',
+            lineThickness: 5,
+            color: '#FF0000',
+          });
+        });
+
+        it('adds a new annotation', async function () {
+          await PageObjects.annotationEditor.addAnnotation({
+            query: 'other query',
+            lineThickness: 3,
+            color: '#00FF00',
+          });
+
+          retry.try(async () => {
+            expect(await PageObjects.annotationEditor.getAnnotationCount()).to.be(2);
+          });
+        });
+
+        it('removes an annotation', async function () {
+          await PageObjects.annotationEditor.removeAnnotation();
+
+          await retry.try(async () => {
+            expect(await PageObjects.annotationEditor.getAnnotationCount()).to.be(1);
+          });
+
+          await PageObjects.annotationEditor.saveGroup();
+          await listingTable.clearSearchFilter();
+        });
+      });
+
+      describe.skip('data view switching', () => {
```

Comment:
This fails in CI but passes locally. I decided to address this as part of https://github.com/elastic/kibana/issues/159053